src/components/Feed/CreatePost.tsx

```diff
@@ -39,6 +48,30 @@ export default function CreatePost() {
     });
   };
 
+  const getUrl = async (file: File | null, postId: Number | any) => {
+    const filename = `${uuidv4()}`;
+    const address = `${env.NEXT_PUBLIC_SUPABASE_URL}/storage/v1/object/public/images/${filename}`;
+    const { data, error } = await supabase.storage
+      .from("images")
+      .upload(filename, file!, {
+        upsert: true,
+      });
+    handleSubmitImage(postId, address);
+    return address;
+  };
+
+  const handleSubmitImage = (postId: number, photoUrl: string) => {
+    console.log(photoUrl);
```

Comment:
Can we remove this console log before we merge?

Reply:
Absolutely.   Great catch.